h/groups/test/views_test.py

```diff
@@ -349,6 +349,125 @@ def test_read_if_already_a_member_returns_response(Group, renderers):
     assert views.read(request) == mock.sentinel.response
 
 
+@read_fixtures
+def test_read_calls_search(Group, search, renderers):
+    """It should call search() to get the annotations."""
+    request = _mock_request(matchdict=_matchdict())
+    g = Group.get_by_pubid.return_value = mock.Mock(slug=mock.sentinel.slug)
+    user = request.authenticated_user = mock.Mock()
+    user.groups = [g]  # The user is a member of the group.
+    renderers.render_to_response.return_value = mock.sentinel.response
+
+    views.read(request)
+
+    search.search.assert_called_once_with(
+            request, private=False, params={"group": g.pubid, "limit": 1000})
+
+
+@read_fixtures
+def test_read_calls_normalize(Group, search, renderers, uri):
+    """It shold call normalize() with each of the URIs from search()."""
```

Comment:
This is literally just supposed to be testing that it calls `normalize()` once for each of the URIs that `search()` returns. But I think you're right that the test below (the one that tests that duplicate normalized URIs are removed) covers this, so this one can just be deleted